AssetManagementTestProject/TestCases/LoginTest.cs

```diff
@@ -7,27 +7,49 @@ namespace AssetManagementTestProject;
 [TestFixture]
 public class LoginTest : NUnitWebTestSetup
 {
+    protected ChangePassword1stTimePage ChangePw1stTime;
+
+    [SetUp]
+    public void SetUp()
+    {
+        ChangePw1stTime = new ChangePassword1stTimePage();
+    }
```

Comment:
Remove setup you can define the value for that variable
`protected ChangePassword1stTimePage ChangePw1stTime = new ChangePassword1stTimePage();`

Reply:
Done

Reply:
Done